packages/react-components/src/components/Dialpad/Dialpad.tsx

```diff
@@ -102,6 +102,8 @@ export interface DialpadProps {
   /**  boolean input to determine if dialpad is in mobile view, default false */
   isMobile?: boolean;
   styles?: DialpadStyles;
+  /* @conditional-compile-remove(dtmf-dialer) */
+  disableDtmfSounds?: boolean;
```

Comment:
Just want to check if we need to be explicit that this still sends the dtmf tone down the phone, it just doesn't play the sound locally - is there a chance a develop thinks this will entirely disable dtmf ?

Reply:
Never mind, then they won't have implemented the onSendDtmfTone

Reply:
Although maybe could consider disableDtmfPlayback instead?

Reply:
this does still send the tone functionally would naming it `disableDtmfAudio` maybe be clearer?